cddl/conditional-series-record.cddl

```diff
@@ -1,6 +1,6 @@
 conditional-series-record = [
   ; reference values to be matched against evidence
-  refv: measurement-values-map
+  refv: measurement-map
```

Comment:
Why is this not [ + measurement-map ] to allow a condition on more than one mkey? I would imagine there are endorsements that are conditional on multiple elements matching expectations and not just one element. Same for adding endorsements.